### ISSUE TITLE
Privacy policy non-compliant 

### ISSUE THREAD

**Issue by lukeswitz** [RelistenNet/relisten-ios#174]
As it stands, the generated policy doesn’t do anything in the way of description of the third party frameworks (dozens). Some of these need proper documentation. 

Fabric/Crashlytics is and has been Google Analytics Firebase for over a year now. Nowhere does it state how long you retain this info on your analytics dashboard settings nor does it offer users any opportunity to have their data delivered to them. 

Listing the libs and frameworks is not enough anymore with GDPR and the various contrasts in the policies of each. You need to explicitly explain why, how, to whom etc and how users can stop the above as well as be delivered theirs. Listing your data protection officer is also a requirement for such data collection practices. If this app is available in EU then there’s a lot more you have to do. 

TLDR: Template generator isn’t cutting it for user privacy information.
